lib/tests/define.rs

```diff
@@ -2118,3 +2118,507 @@ async fn define_statement_table_permissions() -> Result<(), Error> {
 	//
 	Ok(())
 }
+
+#[tokio::test]
+#[cfg(feature = "sql2")]
```

Comment:
Doesn't this test require `sql2`? It doesn't seem to be using `sql2` features.

Reply:
Being new tests doesn't matter @phughk. If they are compatible with `sql1` they will run fine with both. I just ran these queries on v1.1.1 and they run fine and return expected results.

```
#  SurrealDB version: 1.1.1
#
		
ns/db> DEFINE ANALYZER example_blank TOKENIZERS blank;
[NONE]

ns/db> DEFINE ANALYZER example_blank TOKENIZERS blank;
[NONE]

ns/db> INFO FOR DB;
[{ analyzers: { example_blank: 'DEFINE ANALYZER example_blank TOKENIZERS BLANK' }, functions: {  }, models: {  }, params: {  }, scopes: {  }, tables: {  }, tokens: {  }, users: {  } }]
```

Putting this test behind `sql2` isn't necessary.